dojo/engine_tools/helpers.py

```diff
@@ -143,7 +149,6 @@ def expire_finding_exclusion_immediately(finding_exclusion_id: str) -> None:
 @app.task
 def check_expiring_findingexclusions():
```

Comment:
Update the expire_finding_exclusion function to handle the exclusion type, because this function only expires whitelist exclusions.